cyberark/conjur
# Nginx proxy tutorial

Closes CON-4461 (Example exists for using an nginx container to serve Conjur securely with TLS)

#### What does this pull request do?
Provides an example of using Conjur with nginx to provide TLS for Conjur traffic.

#### What background context can you provide?
Background context is provided in `docs/tutorials/integrations/nginx.md`, reproduced in part here:

>  it is crucial to set up TLS correctly when you deploy Conjur...

> Do not leave this to chance: even a small flaw can totally compromise your
> secrets. Setting up Conjur and TLS without the appropriate expertise is like
> packing your own parachute and jumping out of a plane.


#### Where should the reviewer start?
The tutorial flow is in `docs/tutorials/integrations/nginx.md`, the supporting files are in `docs/integrations/tutorials/nginx/`

#### How should this be manually tested?
Run through the tutorial flow and check the container logs to verify that TLS is being used.
